ui/pages/destinations/index.js

```diff
@@ -386,6 +389,11 @@ export default function Destinations() {
                 )}
               </div>
             )}
+            <Pagination
```

Comment:
do we want to hide the pagination if the data is less than 13?

Reply:
so do we want to hide the pagination if there is no next page?

Reply:
Oh i see you put the check on the pagination component. What I would do is probably put it in each parent component - because if there is no next page - we don't need to render the pagination at all